.github/CONTRIBUTING.md

```diff
@@ -16,7 +16,7 @@ If you write and contribute a full topic, we will add your name (or your company
 
 ## Get started
 
-* Make sure you have a [GitHub account](https://github.com/signup/free).
+* Make sure you have a [GitHub account](https://github.com/signup/free). We recommend adding [Two-Factor Authentication](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#two-factor) to your account. Partners are required to add 2FA protection when contributing to Magento respositories.
```

Comment:
I would say "We recommend adding [Two-Factor Authentication](link) (2FA) to your account." Because then you reference 2FA in the following sentence.